app/kubemci/pkg/gcp/loadbalancer/loadbalancersyncer.go

```diff
@@ -411,7 +411,7 @@ func getAnyClient(clients map[string]kubeclient.Interface) (kubeclient.Interface
 		return nil, fmt.Errorf("could not find client to send requests to kubernetes cluster")
 	}
 	// Return the client for any cluster.
-	for k, _ := range clients {
+	for k := range clients {
```

Comment:
what's this change? (well, i'm surprised it compiled before if this wasn't quite right.)

Reply:
We are iterating over a map and ignoring the value.